### PR TITLE
Add python bindings test

### PR DESCRIPTION
**Pull request by l4l**
## What is this pull request?
Test of bindings:
- [x] Python
- [x] Java
  
## How to use the features provided in the pull request?
For python:
`ctest -R python --output-on-failure`
For java:
`ctest -R java --output-on-failure`

### CODE REVIEW

**Comment by neewy** [comment]
As we have recently merged https://github.com/hyperledger/iroha/pull/757 I want to see also tests for true negatives

**Comment by l4l** [comment]
@neewy, actually there's:
https://github.com/hyperledger/iroha/blob/6bc6b7a845586a8ec76db842caa015ad647ddca4/test/module/shared_model/bindings/builder-test.py#L6-L8

**Comment by neewy** [comment]
I mean please check how field validation (stateless) works 🙏

File: test/module/shared_model/bindings/BuilderTest.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    @Test
+    void keygen() {
+        assertEquals(keys.publicKey().size(), 32);
+        assertEquals(keys.privateKey().size(), 64);
+    }
+
+    @Test
+    void addPeer() {
+        UnsignedTx tx = builder.addPeer("123.123.123.123", keys.publicKey()).build();
+        assertEquals(protoSize(tx), 180);
```
Comment (luckychess) [inline test/module/shared_model/bindings/BuilderTest.java]: Looks too easy to break by simple actions like adding new field into transaction proto model. Same for assertions below.

File: test/module/shared_model/bindings/builder-test.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  def proto_size(self, tx):
+    return iroha.ModelTransactionProto().signAndAddSignature(tx, self.keys).size()
+
+  def test_keygen(self):
+    self.assertEqual(len(self.keys.privateKey().blob()), 64)
+    self.assertEqual(len(self.keys.publicKey().blob()), 32)
+
+  def test_add_peer(self):
+    tx = self.builder.addPeer("123.123.123.123", self.keys.publicKey()).build()
+    self.assertEqual(self.proto_size(tx), 180)
```
Comment (luckychess) [inline test/module/shared_model/bindings/builder-test.py]: Same comment as one in Java section.

File: test/module/shared_model/bindings/BuilderTest.java [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+        }
+    }
+
+    long protoSize(UnsignedTx tx) {
+        return new ModelTransactionProto().signAndAddSignature(tx, keys).size();
+    }
+
+    @Test
+    void keygen() {
+        assertEquals(keys.publicKey().size(), 32);
```
Comment (luckychess) [inline test/module/shared_model/bindings/BuilderTest.java]: This could become broken if we change signature algorithm. Do you think this test worth it?

Comment (l4l) [reply test/module/shared_model/bindings/BuilderTest.java]: It works fine until we fix it, yes. I think it's fine to leave it as a regression test

File: test/module/shared_model/bindings/BuilderTest.java (hunk 1 above)
Comment (l4l) [reply test/module/shared_model/bindings/BuilderTest.java]: Yeah, should be fixed to native proto deserialisation test here

**Comment by neewy** [comment]
3 days without any progress with review or more than a single reviewer assigned?

**Comment by l4l** [comment]
I will recreate base on the other base

**Pull request closed without merging**